anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
My change doesn't prevent that at all.  It facilitates it.  In fact it's not actually possible at the moment.  Currently card one would say  _bob your uncle_ for the question and _bob your uncle_ for the answer

Currently anki doesn't care where the sound is in a cloze.  It says them all indiscriminately.

As that gets longer, do you really want to hear the whole thing every time?

You can put whatever audio you want in the hint.  So 
{{c1:: bob[sound:bob]::[sound:is your uncle]}} is
{{c2::your uncle[sound:your uncle]::[sound:bob is]}}
{{c3::mary[sound:mary]::[sound:is your aunt]}} is
{{c4::your aunt[sound:your aunt]::[sound:mary is]}}